riscv/CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+- Improved assembly macro handling in asm.rs
+
+### Added
+
+- Add `miselect` CSR
+
+## [v0.15.0] - 2025-09-08
+
+
```

Comment:
Please accept this suggestion so the changelog is OK:

```suggestion
```

Your changelog file has duplicate sections and shows the same changes more than once. Please, take a look at what it looks like:

<img width="698" height="763" alt="Screenshot 2025-09-17 at 15 54 43" src="https://github.com/user-attachments/assets/912d3fc0-0136-4754-9f53-f99674c656fb" />

Please, for future contributions, make sure that the changelog file looks as it should. You can visually confirm whether it is OK or not.